spec/system/placements/schools/partner_providers/add_a_provider_without_javascript_spec.rb

```diff
@@ -0,0 +1,287 @@
+    and_i_choose_springfield_university
+    and_i_click_on_continue
+    then_i_see_confirm_provider_details_page_for_springfield_university
+
+    # User reconsiders selecting a provider using change link
+    when_i_click_on_change
+    then_i_see_the_search_input_prefilled_with_spring
+    and_i_click_on_continue
+    and_i_choose_springfield_university
+    and_i_click_on_continue
+    then_i_see_confirm_provider_details_page_for_springfield_university
+
+    and_i_click_on_confirm_and_add_provider
+    then_i_return_to_the_provider_index
+    and_i_see_springfield_university_listed
+    and_i_see_success_message_for_springfield_university
+    and_a_notification_email_is_sent_to_springfield_university
+  end
+
+  private
+
+  def given_provider_exists
+    @springfield_university = create(:placements_provider,
+                                     name: "Springfield University",
+                                     ukprn: "10101010",
+                                     urn: "101010",
+                                     email_address: "reception@springfield.ac.uk",
+                                     telephone: "0101 010 0101",
+                                     website: "http://www.springfield.ac.uk",
+                                     address1: "Undisclosed")
+    @springfield_university_user = create(:placements_user, providers: [@springfield_university])
+
+    @ogdenville_university = create(:placements_provider,
+                                    name: "Ogdenville University",
+                                    ukprn: "00100010",
+                                    urn: "000010",
+                                    email_address: "reception@ogdenville.ac.uk",
+                                    telephone: "0000 000 1111",
+                                    website: "http://www.ogdenville.ac.uk",
+                                    address1: "1 Main Street")
+    @ogdenville_university_user = create(:placements_user, providers: [@ogdenville_university])
+
+    @shelbyville_university = create(:placements_provider, name: "Shelbyville University")
+    @shelbyville_college = create(:placements_provider, name: "Shelbyville College")
+    @ogdenville_teacher_training_centre = create(:placements_provider, name: "Ogdenville Teacher Training Centre")
+    @springfield_polytechnic = create(:placements_provider, name: "Springfield Polytechnic")
+  end
+
+  def and_i_am_signed_in
+    @school = create(:school, :placements)
+    given_i_am_signed_in_as_a_placements_user(organisations: [@school])
+  end
+
+  def when_i_am_on_the_providers_index_page
+    visit placements_school_partner_providers_path(@school)
+    expect(page).to have_title("Providers you work with - Manage school placements - GOV.UK")
+    expect(primary_navigation).to have_current_item("Providers")
+    expect(page).to have_h1("Providers you work with")
+  end
+  alias_method :then_i_return_to_the_provider_index, :when_i_am_on_the_providers_index_page
```

Comment:
Same as the comment above https://github.com/DFE-Digital/itt-mentor-services/pull/1181/files#r1848436019